docs/detections/alerts-ui-manage.asciidoc

```diff
@@ -52,25 +52,80 @@ image::images/customize-event-renderer-page.png[Shows the Event Renderer page]
 [float]
 [[view-alert-details]]
 === View alert details
-To further inspect an alert, click the *View details* button from the Alerts table.
+To further inspect an alert, click the *View details* button from the Alerts table. The Alert details flyout appears and offers you several ways to view alert data.
 
 [role="screenshot"]
 image::images/view-alert-details.png[Shows the Event Renderer button, 200]
 
-The Alert details flyout appears and offers several options for viewing alert details:
+Each of the tabs within the Alert details flyout are described below.
 
-* *Summary*: Shows an aggregated view of alert details. Alerts that have been enriched with `threat.indicator` data also display the *threat summary* section, which is an additional section located beneath the alert summary. In the *threat summary* section, you can view mapped data for the following `threat.indicator` subfields:
-** `matched.field`
-** `matched.type`
-** `source (threat.indicator.provider)`
-** `first_seen`
-** `last_seen`
+[discrete]
+*Summary tab*
+
+The *Summary* tab provides an overview of the alert and shows relevant threat intelligence details. You can use this information to gain an understanding of what caused the alert and to make decisions about handling it.
+
+The *Summary* tab always begins with a list of general alert details. These details are conveyed as a curated list of fields containing essential and available information about the alert. Within this section, you can find the alert's current status, locate the rule that produced the alert, learn the alert's severity, and more. You can use these details to start forming a sense of what caused the alert and then continue your investigation by checking out available threat intelligence on the alert.
+
+The {es-sec} app collects threat intelligence data for all alerts. Fields that show threat intelligence data are appended with an informational icon and placed after general alert details on the *Summary* tab.
+
+When gathering threat intelligence, the {es-sec} app  queries indicator data from the past 30 days to generate enrichments for the below fields.
```

Comment:
I would avoid the verb `generate` as that could be interpreted as generating _persistent_ data, which this is not:

```suggestion
When gathering threat intelligence, the {es-sec} app queries indicator data from the past 30 days to find enrichments for the below fields.
```

Reply:
I would also avoid the `enrichments` term for now:
```suggestion
When gathering threat intelligence, the {es-sec} app queries indicator data from the past 30 days to find threat intelligence for the below fields.
```